tests/test_functional.py

```diff
@@ -285,6 +285,49 @@ def test_legend_caching(pcic_data_portal):
     req.if_modified_since = pre_load_time
     resp = req.get_response(pcic_data_portal)
     assert resp.status.startswith('200')
+    
+
+@pytest.mark.crmpdb
+@pytest.mark.parametrize('polygon', ['single station polygon', 
```

Comment:
Cool, great tests! This will be nice to have all of these weird corner cases in the test suite.